Module/templates/bug-report.html

```diff
@@ -61,6 +66,28 @@ <h4 class="flexrow">
     </div>
   </div>
 
+  <div class="{{#if (bugs-isEmpty conflicts)}}hidden{{/if}}">
+    <label>This module is known to conflict with the following:
+    <ul>
+    {{#each conflicts as | conflict |}}
```

Comment:
`as | conflict |` is unecessary, doesn't hurt anything but the scope of 'this' becomes what you're assigning to `conflict` by default within the `#each`. Could simplify this after removing the `as` by:
- `conflict.active` -> `active` 
- `conflict.name` -> `name`
- etc...

Reply:
Huh, didn't know that!